cmd/tempo/app/app.go

```diff
@@ -62,7 +62,7 @@ func (c *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet) {
 	c.Target = All
 	// global settings
 	f.StringVar(&c.Target, "target", All, "target module")
-	f.BoolVar(&c.AuthEnabled, "auth.enabled", true, "Set to false to disable auth.")
+	f.BoolVar(&c.AuthEnabled, "auth.enabled", false, "Set to true to enable auth.")
```

Comment:
@grafana/tempo should we take this as an opportunity to improve this setting name?

`multitenancy.enabled` ?

Reply:
+1

Reply:
@dgzlopes Do you mind taking this on?

I think ideally we would actually support both and just mark `auth.enabled` deprecated and remove it from docs. A lot of people are going to have this setting in their config files.

Reply:
Yup, I'll refactor the PR :)

Reply:
Done! I added the new setting, marked the old one as deprecated, and removed a bunch of references to it.